src/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx

```diff
@@ -78,110 +75,116 @@ export const ArtistAuctionResultItem: React.FC<Props> = props => {
       width="100%"
       onClick={onAuctionResultClick}
     >
-      <GridColumns>
-        <Column span={[4, 2]}>
-          <ResponsiveBox
-            aspectWidth={1}
-            aspectHeight={1}
-            maxWidth={130}
-            bg="black10"
-          >
-            {image ? (
-              <Image
-                src={image.src}
-                srcSet={image.srcSet}
-                width="100%"
-                height="100%"
-                alt=""
-                lazyLoad
-              />
-            ) : (
-              <Box
-                display="flex"
-                justifyContent="center"
-                alignItems="center"
-                height="100%"
-              >
-                <NoArtIcon height={24} width={24} fill="black60" />
-              </Box>
-            )}
-          </ResponsiveBox>
-        </Column>
+      <RouterLink
+        to={`/auction-result/${internalID}`}
+        textDecoration="none"
+        style={{ pointerEvents: user ? "auto" : "none" }}
```

Comment:
Great solution!

**Nitpick:**
But I tested it with keyboard navigation and managed to get to the screen without being logged in

Screen recording: <video src="https://user-images.githubusercontent.com/20655703/235910503-40209486-cc1d-46b5-8d0a-d88747837897.mov" /> 


My suggestion is either use [`tabIndex`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex) as -1 so the keyboard doesn't focus on it (affects keyboard accessibility) or have the data hidden in the Auction Result screen if the user is not logged in

Using tabIndex example
```tsx
...
tabIndex={user ? 0 : -1}
...
```

Reply:
good catch @MrSltun and thanks for the suggestion.
@dzucconi do you have suggestions on how we can add this href without making it tappable? And if actually we don't have to do that. Thank you